jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -1080,14 +1112,44 @@ public void setStrictEventOrdering(boolean strictEventOrdering)
         this.strictEventOrdering = strictEventOrdering;
     }
 
+    /**
+     * @return the delay in ms before idle destinations should be removed
+     * @see #setDestinationIdleTimeout(long)
+     */
+    @ManagedAttribute("The delay in ms before idle destinations are removed, disabled when zero or negative")
+    public long getDestinationIdleTimeout()
+    {
+        return destinationIdleTimeout;
+    }
+
+    /**
+     * Whether destinations that have no connections (nor active nor idle) should be removed
+     * after a certain delay.
+     * <p>
+     * Applications typically make request to a limited number of destinations so keeping
+     * destinations around is not a problem for the memory or the GC.
+     * However, for applications that hit millions of different destinations (e.g. a spider
+     * bot) it would be useful to be able to remove the old destinations that won't be visited
+     * anymore and leave space for new destinations.
+     *
+     * @param destinationIdleTimeout the delay in ms before idle destinations should be removed
+     * @see org.eclipse.jetty.client.DuplexConnectionPool
```

Comment:
Why this reference? I think it's unrelated.

Reply:
Done.